security.html

```diff
@@ -0,0 +1,127 @@
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+            <a href="#navigation">Skip to Navigation</a>
+        </div>
+        
+        <div class="floe">
+            <div class="floe-header">
+                <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </div>
+
+            <div class="floe-content floe-video">
+                <div class="flc-toc-tocContainer toc"> </div>
+                <div class="floe-breadcrumbs"><a href="index.html">&lt; back</a></div>
+                <h2> Security and Privacy Architecture </h2>
+    
+                <p>The Floe Project is working with the
+                    <a href="http://gpii.net">Global Public Inclusive Infrastructure</a>
+                    to provide learners with friendly and comprehensive tools for sharing their data
+                    with the web sites, access features and assistive technologies that they trust.
+                    These tools will give learners complete control over which preferences are shared
+                    with any GPII-enabled services.</p>
+                <p>For more information and to see an example of the security architecture in action,
+                    please see the video below.</p>
+                <video controls width="640px" height="480px" tabindex="0">
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.mp4" type="video/mp4"></source>
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.webm" type="video/webm"></source>
+                </video>
+            </div>
+        
+            <div class="floe-footer">
+                <div class="floe-content clearfix">
+                    <div class="floe-column">
+                        Floe is a project of the
+                        <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>, funded by a grant from
+                        <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.
+                        <br/>
+                        <a class="floe-image-link" href="http://www.hewlett.org/"> <img alt="Hewlett foundation logo" src="images/hewlett_foundation_logo.png"/> </a>
+                    </div>
+                    
+                    <div class="floe-column">
+                        <p>
+                            205 Richmond St. W., Second Floor<br/>
+                            Toronto, Ontario, Canada<br/>
+                            M5V 1V3<br/>
+                            Telephone: (416) 977-6000 #3968<br/>
+                            Fax: (416) 977-9844
+                        </p>
```

Comment:
Could we put the address in an `<address>` element?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address